webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityStreamToolbar.vue

```diff
@@ -95,10 +95,11 @@ export default {
   },
   methods: {
     openComposerDrawer() {
-      const storageMessage = localStorage.getItem('activity-message') || '';
+      const storageMessage =  JSON.parse(localStorage.getItem('activity-message'));
+      const storageMessageText = (storageMessage?.url && storageMessage.url === eXo.env.server.portalBaseURL) ? storageMessage.text : '';
```

Comment:
You may need to check if the localStorage value is empty or not.
```suggestion
      const storageMessage =  localStorage.getItem('activity-message');
      const storageMessageObject =  storageMessage && JSON.parse(storageMessage) || {};
      const storageMessageText = storageMessage?.url === eXo.env.server.portalBaseURL && storageMessage?.text || "";
```